src/Compilers/CSharp/Test/Semantic/Semantics/RawInterpolationTests_Handler.cs

```diff
@@ -3727,7 +3727,7 @@ .locals init (S V_0)
 ");
     }
 
-    [Theory]
+    [ConditionalTheory(typeof(IsEnglishLocal))]
```

Comment:
>(typeof(IsEnglishLocal) [](http://example.com/codeflow?start=22&length=23)

In general, we prefer to not skip tests in this manner. We also would like have a clear understanding of the issue because the test failure might indicate a product issue rather than a test issue.

Reply:
If it is determined that the behavior is expected to be localization dependent, there are ways to force the test to use specific language. We prefer that approach over simply disabling a test.